frontend/jupyter/src/app/pages/volume-form/volume-form.component.html

```diff
@@ -52,9 +52,9 @@
     <mat-form-field appearance="outline" class="wide">
       <mat-label i18n>Access Mode</mat-label>
       <mat-select formControlName="mode">
-        <mat-option value="ReadWriteOnce" i18n>ReadWriteOnce</mat-option>
-        <mat-option value="ReadOnlyMany" i18n>ReadOnlyMany</mat-option>
-        <mat-option value="ReadWriteMany" i18n>ReadWriteMany</mat-option>
+        <mat-option value="ReadWriteOnce">ReadWriteOnce</mat-option>
+        <mat-option value="ReadOnlyMany">ReadOnlyMany</mat-option>
+        <mat-option value="ReadWriteMany">ReadWriteMany</mat-option>
```

Comment:
Why remove the translations for those? Unless they were never really translated, but it was done in a 'eventually we will need good translation' idea.